master/internal/task/allocation.go

```diff
@@ -205,7 +201,6 @@ func (a *Allocation) Receive(ctx *actor.Context) error {
 	// These messages allow users (and sometimes an orchestrator, such as HP search)
 	// to interact with the allocation. The usually trace back to API calls.
 	case AllocationReady:
-		a.model.State = &a.state // TODO(Brad): Consolidate where state is store else bugs.
```

Comment:
I was slightly perplexed by this line: did it do anything?

Reply:
no, it didn't